services-js/public-notices/src/components/NoticeList.stories.tsx

```diff
@@ -8,9 +8,12 @@ import NoticeList from './NoticeList';
 const NOTICES: Notice[] = require('../../fixtures/notices.json');
 
 storiesOf('NoticeList', module).add('default', () => (
-  <div style={{ width: 450, backgroundColor: 'white' }}>
+  // This width and height match the column when it’s on a TV.
```

Comment:
Good tidbit of explanation on where those values came from. Are they valid for _any_ TV, or one model in particular? If the latter, we should add that addt’l info to the comment.

Reply:
Updated comment to specify 1080p TV.